lya/data.py

```diff
@@ -1,43 +1,73 @@
 from .conf import lya_data_path
-
+import numpy as np
+import matplotlib.pyplot as plt
 
 class Data(object):
-    def __init__(self, x, y, y_unc):
-        self.x = x
-        self.y = y
-        self.y_unc = y_unc
 
+    def __init__(self,
+                 k=None,
+                 P_k=None,
+                 P_k_unc=None,
+                 ):
 
-def load_lya_data(filename=lya_data_path):
-    """Load lyman-alpha data
+        self.k = k
+        self.P_k = P_k
+        self.P_k_unc = P_k_unc
```

Comment:
This is confusing since we also use the Data object to store the transfer function x/y/uncertainty values in `get_data_transfer_function`, so this change will break PR #14. Can we keep the variable names general (x, y, y_unc)?

Reply:
Yeah, the `get_data_transfer_function` should also become part of the Data class. I suggest we assign k, Pk, Tk etc as Data attributes. Then we can use the data class nicely like this: 

```python
data = Data()
data.load_lya_txt()
```
now data has the attributes we want and can be called with `data.k`, `data.P_k`, etc.
If we do the same thing with the model, we can also have `model.k`, `model.P_k` etc.  which would be nice. 

with this, we can add a `.get_transfer_function` method to data:

```
data.get_transfer_function(model)
```
 which can use `model.P_k` and calculate `data.T_k`. 

Does this make sense?